DataFormats/HcalDigi/interface/HcalUMNioDigi.h

```diff
@@ -0,0 +1,46 @@
+#ifndef DATAFORMATS_HCALDIGI_HCALUMNIODIGI_H
+#define DATAFORMATS_HCALDIGI_HCALUMNIODIGI_H 1
+
+#include <vector>
+#include <stdint.h>
+#include <ostream>
+
+/** \class HcalUMNioDigi
+  *  
+  * $Date: $
+  * $Revision: $
+  * \author J. Mans - Minnesota
+  */
+class HcalUMNioDigi {
```

Comment:
The electronics card which reads the laser ampltiude is the "uTCA University of Minnesota I/O card with QIE daughter card" which is abbreviated as "uMNqie".  The "u" stands for "uTCA".